.env.example

```diff
@@ -9,3 +9,11 @@
 
 # DIRECT_URL is given to you if using Neon. If using another postgres instance, set it to the same value as DATABASE_URL
 # DIRECT_URL=postgres://postgres:123456@127.0.0.1:5432/dummy
+
+# NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=pk_test_c2FmZS1yYXZlbi02OC5jbGVyay5hY2NvdW50cy5kZXYk
+# CLERK_SECRET_KEY=
+
+# NEXT_PUBLIC_CLERK_SIGN_IN_URL=/
+# NEXT_PUBLIC_CLERK_SIGN_UP_URL=/
+# NEXT_PUBLIC_CLERK_AFTER_SIGN_IN_URL=/
+# NEXT_PUBLIC_CLERK_AFTER_SIGN_UP_URL=/
```

Comment:
are these env variables necessary / used? what is their purpose?

Reply:
These are the default redirect URLs used by clerk after completing signin/signup, not sure if removing them would change anything but let's leave them in for completeness even if we don't need them